src/github.com/matrix-org/dendrite/cmd/dendrite-clientapi/main.go

```diff
@@ -52,9 +69,9 @@ func main() {
 		ServerName:           "localhost",
 		KeyID:                "ed25519:something",
 		PrivateKey:           privKey,
-		KafkaProducerURIs:    []string{"localhost:9092"},
-		ClientAPIOutputTopic: "roomserverInput",
-		RoomserverURL:        "http://localhost:7777",
+		KafkaProducerURIs:    kafkaURIs,
+		ClientAPIOutputTopic: clientAPIOutputTopic,
```

Comment:
Hmm, I really need to come up with better names for these.
Ideally they should be the same environment variable for all processes.